_posts/blog/2016-12-22-dwh-part1_getting_started.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: "A journey towards a custom data warehouse solution Part 1 : Getting Started"
+description:
+modified:
+categories: blog
+author: tino_noack
+excerpt: This is part one of a series of articles in which we would like to share our journey with you and our experiences towards a custom data warehouse solution. Part one starts with a brief introduction to upday including the kinds of data we work with and a description of our working environment. Furthermore, we come up with a tentative definition of a data warehouse (DWH), describe components regarding our use case, and identify tools which can be applied for implementation. Finally, we introduce a data lake as a more 'Big-Data-friendly' but provisional solution.
+tags: [Data, DWH, Data Warehouse, Big Data, Data Platform, Data Lake]
+image:
+date: 2016-12-22T00:00:00-04:00
+---
+# Abstract
+
+This is part one of a series of articles in which we would like to share our journey with you and our experiences towards a custom data warehouse solution. Part one starts with a
+brief introduction to upday including the kinds of data we work with and a description of our working environment. Furthermore, we come up with a tentative definition of a data
+warehouse (DWH), describe components regarding our use case, and identify tools which can be applied for implementation. Finally, we introduce a data lake as a more
+'Big-Data-friendly' but provisional solution.
+
+# Getting Started
+
+upday is a news app which aggregates content from thousands of sources including trusted brands and bloggers into a unique and simple format. It is available in a variety of
+European countries in a wide range of languages. upday is designed to be an intelligent news aggregator which learns about the reading habits and preferences of each unique user.
+For that reason, we have to cope with a huge volume of incoming data from a vast number of sources. For example, we handle the following kinds of data:
+
+1. _Content metadata_ such as information about the content type, the publisher, and the publishing date is amongst others used for maintaining the publisher relationship. Content
+   metadata is semi-structured but also contains unstructured data such as a title, a description, and the content itself e.g. in text form.
+2. _User preferences_ are reading interests explicitly selected by the user and are used to personalize the content we deliver to the user. User preferences data is a form of
+   structured data.
+3. _User behavior_ includes information about the content consumed by the user and is used to improve the users’ experience with our product for example by means of collaborative
```

Comment:
can you mention that the data is annonymous, that we can't identify the person based on that, and that it's not sold. 
Maybe rephrase this in a nice way to make all of these understood.